src/components/map/LocationOverview.tsx

```diff
@@ -180,10 +177,10 @@ const styles = StyleSheet.create({
 
 /* eslint-disable react-native/no-unused-styles */
 /* this works properly, we do not want that warning */
-const stylesWithProps = ({ navigation }: { navigation: string }) => {
+const stylesWithProps = ({ navigationType }: { navigationType: string }) => {
   return StyleSheet.create({
     position: {
-      bottom: navigation === 'tab' ? '4%' : '8%'
+      bottom: navigationType === 'tab' ? '4%' : '8%'
     }
   });
 };
```

Comment:
I have tried different navigation types. If no navigation value is specified in `globalSettins`, the following screenshot appears. Can you please check if this is the case with you? It is surprising that the preview looks like this even though we set the `navigationType` to `tab` by default.

|navigation = "tab"|navigation = undefined|navigation = "drawer"|
|--|--|--|
![Simulator Screenshot - iPhone 14 Pro Max - 2023-10-13 at 09 41 45](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/4751c6fc-9631-4949-b6ec-53ea774eac15)|![Simulator Screenshot - iPhone 14 Pro Max - 2023-10-13 at 09 41 10](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/f99362aa-96d9-4a0f-bbd1-c3c6c8b31152)|![Simulator Screenshot - iPhone 14 Pro Max - 2023-10-13 at 09 42 08](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/785648d5-3eb4-41bd-a76b-771c90920877)

Reply:
🤔 i will look into it